flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/source/enumerator/KafkaSourceEnumStateSerializer.java

```diff
@@ -113,23 +111,26 @@ public KafkaSourceEnumState deserialize(int version, byte[] serialized) throws I
         }
     }
 
-    private static Set<TopicPartition> deserializeTopicPartitions(byte[] serializedTopicPartitions)
+    private static KafkaSourceEnumState deserializeTopicPartitions(byte[] serializedTopicPartitions,
+                                                                   AssignmentStatus status)
```

Comment:
I'd rename to `deserializeAssignedPartitions` and inline `status`. I don't see it being used with `UNASSIGNED`.

Reply:
> I'd rename to `deserializeAssignedPartitions` and inline `status`. I don't see it being used with `UNASSIGNED`.

Thank you for your careful review, I changed to them.

Reply:
> I'd rename to `deserializeAssignedPartitions` and inline `status`. I don't see it being used with `UNASSIGNED`.

done.